front/src/modules/pathfinding/components/Itinerary/Itinerary.tsx

```diff
@@ -49,6 +50,8 @@ const Itinerary = ({
   const { t } = useTranslation('operationalStudies/manageTrainSchedule');
   const { openModal } = useModal();
 
+  const isPathStepInvalid = (step: PathStep | null): boolean => step?.isInvalid || false;
```

Comment:
Do you think you can move this function in a utils ?

Reply:
done